lib/ec2imgutils/ec2disableimgdeprecation.py

```diff
@@ -0,0 +1,198 @@
+# Copyright 2025 SUSE LLC
+#
+# This file is part of ec2imgutils
+#
+# ec2imgutils is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ec2imgutils is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ec2deprecateimg. If not, see <http://www.gnu.org/licenses/>.
+
+import logging
+
+import ec2imgutils.ec2utils as utils
+from ec2imgutils.ec2imgutils import EC2ImgUtils
+from ec2imgutils.ec2imgutilsExceptions import EC2DisableImgDeprecationException
+
+
+class EC2DisableImgDeprecation(EC2ImgUtils):
```

Comment:
This is not `disabling` it `is removing`.

Reply:
The Amazon name for it is `disable_image_deprecation` https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2/client/disable_image_deprecation.html

Having it as part of the already existing deprecate command I think is okay too. This is how Google's API is setup. The Google Deprecate command can be used to set images back to active.

Reply:
Yes, I forgot about the bad choice of terms on the AWS side with `enable_image_deprecation` which is really `deprecate_image` and as such the the unfortunate choice of `disable`. That said I don't think we have to follow suit in choosing bad names.